src/main/features/response/views/your-dob.njk

```diff
@@ -3,17 +3,18 @@
 {% from "date.njk" import dateInput %}
 
 {% set title = t('Money claim') %}
-{% set heading = t('What is your date of birth?') %}
+{% set heading = t('Add your date of birth') %}
 
 {% block content %}
   <div class="grid-row">
     <div class="column-two-thirds">
+      <p>{{ t('We use your date of birth to make sure you’re over 18 and that legal proceedings involving a claim are taken against the right person.') }}</p>
```

Comment:
could you break this into multiple lines please?
less scrolling is better for re

Reply:
done